src/lib/config.js

```diff
@@ -47,6 +47,22 @@ function parseCredentials () {
   }, {})
 }
 
+function parseAdminEnv () {
+  const adminUser = Config.getEnv(envPrefix, 'ADMIN_USER') || 'admin'
```

Comment:
Why do we need to default to `admin`? A real, non-default user name should always be specified I'd think.